server/events/models/models.go

```diff
@@ -491,6 +492,16 @@ type PlanSuccess struct {
 	HasDiverged bool
 }
 
+// Summary extracts one line summary of plan changes from TerraformOutput.
+func (p *PlanSuccess) Summary() string {
+	r := regexp.MustCompile(`Plan: \d+ to add, \d+ to change, \d+ to destroy.`)
+	if match := r.FindString(p.TerraformOutput); match != "" {
+		return match
+	}
+	r = regexp.MustCompile(`No changes. Infrastructure is up-to-date.`)
```

Comment:
Just confirming, this won't even be called if there is a plan error right? We don't want it to say `No changes` when a plan errors out.